crates/l2/contracts/src/l1/CommonBridge.sol

```diff
@@ -84,16 +97,16 @@ contract CommonBridge is
         return pendingDepositLogs;
     }
 
-    function _deposit(DepositValues memory depositValues) private {
-        require(msg.value > 0, "CommonBridge: amount to deposit is zero");
+    function _deposit(address from, DepositValues memory depositValues) private {
+        depositsERC20[ETH_TOKEN][ETH_TOKEN] += msg.value;
```

Comment:
This call is confusing since it's also being called on non-ETH deposits. We should move it to the caller's side